CHANGELOG.md

```diff
@@ -3,10 +3,16 @@
 Subscribe to the [tracking issue #2231](https://github.com/tigerbeetle/tigerbeetle/issues/2231)
 to receive notifications about breaking changes!
 
-## TigerBeetle 0.16.24
+## TigerBeetle 0.16.25
 
 Released: 2025-01-27
 
+## TigerBeetle (unreleased)
+
+Released: 2025-01-27
+
+(Unreleased due to CI flake.)
+
```

Comment:
```suggestion
```

I _think_ you just need to bump the version number in the changelog, as per https://github.com/tigerbeetle/tigerbeetle/blob/main/docs/about/internals/releases.md#error-handling

Reply:
:grimacing: I copy-pasted from https://github.com/tigerbeetle/tigerbeetle/pull/2658!